design-system-docs/frontend/styles/index.scss

```diff
@@ -1,4 +1,4 @@
-@import '~@citizensadvice/design-system/scss/lib';
+@import '../scss/lib';
```

Comment:
This feels like too few `..` but the sass-loader seems to resolve from the root of the project, so this works as expected.